### PR TITLE
Bump go 1.20.5 in line  0.35.x

### PR DESCRIPTION
**Pull request by kumaritanushree**

### CODE REVIEW

File: hack/Dockerfile.dev [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM golang:1.19
+FROM golang:1.20
 
 RUN apt-get update -y
-RUN apt-get install docker.io apt-transport-https ca-certificates gnupg python -y
+RUN apt-get install docker.io apt-transport-https ca-certificates gnupg python-is-python3 -y
```
Comment (praveenrewar) [inline hack/Dockerfile.dev:4]: Don't we want to make this change in develop first?

**Review by praveenrewar — COMMENTED** [review]
Quick question ^

File: hack/Dockerfile.dev (hunk 1 above)
Comment (joaopapereira) [reply hack/Dockerfile.dev:4]: The change was already merged in develop

**Review by joaopapereira — APPROVED** [review]
LGTM

**Pull request merged**